docs/commands.rst

```diff
@@ -7,9 +7,9 @@ Commands
 ========
 
 
-Every commands in ``vagga.yaml`` is mapping with a tag that denotes command
-type. The are two command types ``!Command`` and ``!Supervise`` illustrated
-by the following example:
+Every command under ``commands`` in ``vagga.yaml`` is mapped with a tag
```

Comment:
Well, I'm not sure either. If you find better wording, please send a pull request.